org.eclipse.winery.frontends/app/topologymodeler/src/app/models/InheritanceUtils.ts

```diff
@@ -107,6 +107,21 @@ export class InheritanceUtils {
         return listOfEffectiveRequirementDefinitions;
     }
 
+    static getDescendantsOfEntityType<T extends EntityType>(entityType: string, entityTypes: T[]): T[] {
```

Comment:
Please add a comment on the method, that the returned array has the `entityType` being queried at position 0, and is not ordered otherwise.
Alternatively, find a way to ensure that the resulting elements are ordered based on their "distance" from the `entityType`.